### ISSUE TITLE
Some question about the results of other works in the thesis

### ISSUE THREAD

**Issue by pekingtong** [DearCaat/RRT-MIL#1]
Nice work, The results of other papers in the article are higher than what they mentioned in their article. Can you provide the seeds of data division or training if possible?

**Comment by DearCaat** [comment]
Thanks for attention.

For CAMELYON-16, we used `2021, 2022, 2023` three seeds. For other datasets, we used `2021` seed.
Note that, instead of validating directly with the official test set, we used `three-times 3-fold cross-validation`. This is a different setup than the original article (e.g., TransMIL, DTFD-MIL) and may be a reason for the different results. 

In addition to that, since CAMELYON-16 has only 398 slides (TCGA-NSCLS ~ 1000 slides), arbitrary random factors or uncontrollable other factors can affect the final results more significantly. Therefore, we did not complete the experiment with the goal of completely reproducing the results of the previous work.
